delphi-frontend/src/main/java/au/com/integradev/delphi/symbol/declaration/QualifiedNameDeclarationImpl.java

```diff
@@ -30,13 +32,18 @@ public abstract class QualifiedNameDeclarationImpl extends NameDeclarationImpl
   private final QualifiedName qualifiedName;
 
   protected QualifiedNameDeclarationImpl(QualifiedNameDeclarationNode node) {
-    super(node);
-    qualifiedName = node.getQualifiedName();
+    super(node, node.getScope());
+    this.qualifiedName = node.getQualifiedName();
```

Comment:
Is this change necessary? It looks like it's functionally equivalent to how it was before.

Reply:
Oops, artifact of a change I almost made and later decided against.
I'll revert that.